firestore/src/main/create_firebase_metadata_provider.h

```diff
@@ -1,3 +1,5 @@
+// Copyright 2021 Google LLC
+
 #ifndef FIREBASE_FIRESTORE_CLIENT_CPP_SRC_MAIN_CREATE_FIREBASE_METADATA_PROVIDER_H_
```

Comment:
⚠️ Lint warning: `Lines should be <= 80 characters long`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/src/main/create_firebase_metadata_provider.h:3:  Lines should be <= 80 characters long  [whitespace/line_length] [2]"></hidden>